### ISSUE TITLE
Accessibility language issue: aria label of the nav button is not follow the locale language

### ISSUE THREAD

**Issue by theolemague** [gpbl/react-day-picker#2798]
When giving a locale props for the language settings (as example fr from `date-fns/locale/fr`) the aria-label navigation button the select the month are not translated in the right language are still in English always (same behaviour see the [Persian calendar](https://daypicker.dev/docs/localization#persian-calendar) for exemple 

## Code

```tsx
import fr from "date-fns/locale/fr";
import { DayPicker } from "react-day-picker";
import "react-day-picker/style.css";

export default function App() {
  return <DayPicker mode="single" locale={fr}/>;
}
```

## HTML

```
<nav class="rdp-nav" aria-label="">
   <button type="button" class="rdp-button_previous" aria-label="Go to the Previous Month">...</button> // should be "aller au mois précédent"
   <button type="button" class="rdp-button_next" aria-label="Go to the Next Month">...</button>
</nav>
```
